saritasa_invocations/alembic.py

```diff
@@ -0,0 +1,147 @@
+    message="",
+) -> None:
+    """Autogenerate new version with `message` title.
+
+    Similar to the `manage.py makemigration` command in Django. But you always
+    have to check generated versions before upgrade and fix it manually
+    if it's necessary.
+
+    """
+    if not message:
+        raise invoke.Exit(
+            code=1,
+            message=(
+                'Please, use `-m "Version message"` to set message '
+                "for autogenerated version."
+            ),
+        )
+    printing.print_success("Autogenerate migrations")
+    config = context.config.get("migrations_folder", {})
+    migrations_folder = config.get(
+        "migrations_folder",
+        "db/migrations/versions",
+    )
+    rev_id = str(
+        len(
+            [
+                name
+                for name in os.listdir(migrations_folder)
+                if os.path.isfile(f"{migrations_folder}/{name}")
+            ],
+        )
+        + 1,
+    )
+    rev_id = rev_id.rjust(4, "0")
+    command = (
+        f'revision --autogenerate --message "{message}" --rev-id={rev_id}'
+    )
+    run(context, command=command)
+
+
+@invoke.task
+def upgrade(
+    context: invoke.Context,
+    version: str = "head",
+) -> None:
+    """Upgrade database.
+
+    Use `-v version` to upgrade to the passed version.
+
+    Similar to the `manage.py migrate` command in Django. By default upgrade to
+    the `head` - the latest version.
+
+    """
+    printing.print_success(f"Migrating to version {version}")
+    run(context, command=f"upgrade {version}")
+
+
+@invoke.task
+def downgrade(
+    context: invoke.Context,
+    version: str = "base",
+) -> None:
+    """Downgrade database.
+
+    Use `-v version` to downgrade to the passed version.
+
+    Similar to the `manage.py migrate` command in Django. By default downgrade
+    to `base` (similar to `migrate zero`).
+
+    """
+    printing.print_success(f"Migrating to version {version}")
+    run(context, command=f"downgrade {version}")
+
+
+@invoke.task
+def check_for_migrations(
+    context: invoke.Context,
+) -> None:
+    """Check if new migration can be generated."""
+    printing.print_success("Checking for migration")
+    run(context, command="check")
+
+
+@invoke.task
+def check_for_adjust_messages(
+    context: invoke.Context,
+) -> None:
+    """Check migration files for adjust messages."""
+    printing.print_success(
+        "Checking migration files for adjust messages",
+    )
+    config = context.config.get("migrations_folder", {})
+    migrations_folder = config.get(
+        "alembic_migrations_folder",
+        "db/migrations/versions",
+    )
+    adjust_messages = config.get(
+        "alembic_adjust_messages",
+        (
+            "# ### commands auto generated by Alembic - please adjust! ###",
+            "# ### end Alembic commands ###",
+        ),
+    )
+    files_to_clean = collections.defaultdict(set)
+    for filename in os.listdir(migrations_folder):
```

Comment:
Let's use `pathlib` here and below (start with https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob)